examples/map_range_demo.py

```diff
@@ -0,0 +1,22 @@
+"""
+'map_range_demo.py'.
+
+=================================================
+maps a number from one range to another
+"""
+import time
+import simpleio
+
+while True:
+    SENSOR_VALUE = 150
```

Comment:
The variables should be lower case. Travis runs pylint on examples with invalid-name off for this reason: https://github.com/adafruit/Adafruit_CircuitPython_SimpleIO/blob/master/.travis.yml#L29

Reply:
Ok, variables switched to lower case.